cnf-certification-test/operator/suite.go

```diff
@@ -317,3 +347,100 @@ func testOperatorSingleCrdOwner(check *checksdb.Check, env *provider.TestEnviron
 
 	check.SetResult(compliantObjects, nonCompliantObjects)
 }
+
+func testOperatorPodsRunAsUserID(check *checksdb.Check, env *provider.TestEnvironment) {
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	for csv, pods := range env.CSVToPodListMap {
+		check.LogInfo("Csv: %q", csv)
+		for _, pod := range pods {
+			check.LogInfo("Testing Pod %q", pod)
+			if pod.IsRunAsUserID(0) {
+				check.LogError("Pod %q UserID of the pods created by the operator is 0", pod.Name)
```

Comment:
Better alternative is : 
`check.LogError("Pod %s in namespace %s is running with UserID = 0", pod.Name, pod.Namespace)`

Reply:
i agree , i will fix all the logs.